service_container/factories.rst

```diff
@@ -293,5 +293,138 @@ previous examples takes the ``templating`` service as an argument:
                 ->args([ref('templating')])
             ;
         };
+        
+Usage example
+--------------------
```

Comment:
Underline to long

Reply:
Thanks for advise, changes have been introduced.